### PR TITLE
User registration feature file

### PR DESCRIPTION
**Pull request by susmitathapa**
### Description
In this feature file I have written different scenarios for user registration feature.

### Related Issue
https://github.com/SagarGi/FeatureFiles/issues/2

### CODE REVIEW

File: Features5/tests/features/newUserRegistration.feature [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+   Background:
+       Given user has been navigated to registration page
+   
+   Scenario: New user registration
+       When user enters following valid credentials:
+         |name    |email                |password   |confirmPassword  |
+         |Susmita |susmita412@gmail.com |zxcvbnm    |zxcvbnm          |
+         |Sita    |sita123@gmail.com    |abcdef     |abcdef           |
+       And clicks "Register" button
```
Comment (kiranadh1452) [inline Features5/tests/features/newUserRegistration.feature]: The `register` button is clicked after filling both the user's data. How would the first user be registered?

**Review by kiranadh1452 — CHANGES_REQUESTED** [review]

File: Features5/tests/features/newUserRegistration.feature [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+   Background:
+       Given user has been navigated to registration page
+   
+   Scenario: New user registration with valid inputs
+       When user enters following valid credentials:
+         |name    |email                |password   |confirmPassword  |
+         |Susmita |susmita412@gmail.com |zxcvbnm    |zxcvbnm          |
+       And clicks "Register" button
+       Then user should see the homepage
```
Comment (SagarGi) [inline Features5/tests/features/newUserRegistration.feature:14]:
```suggestion
       Then user should be able see the homepage
```

File: Features5/tests/features/newUserRegistration.feature [hunk 3]
```diff
@@ -0,0 +1,27 @@
+Feature: Register
+   As a user
+   I want to register
+   So that I can use the contact fox app
+
+   Background:
+       Given user has been navigated to registration page
+   
+   Scenario: New user registration with valid inputs
+       When user enters following valid credentials:
```
Comment (SagarGi) [inline Features5/tests/features/newUserRegistration.feature:10]:
```suggestion
       When user enters following valid credentials using webUI:
```

File: Features5/tests/features/newUserRegistration.feature [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+         |name    |email                |password   |confirmPassword  |
+         |Susmita |susmita412@gmail.com |zxcvbnm    |zxcvbnm          |
+       And clicks "Register" button
+       Then user should see the homepage
+
+    Scenario Outline: When new user registers with invalid inputs
+       When user enters "<name>","<email>","<password>" and "<confirmPassword>"
```
Comment (SagarGi) [inline Features5/tests/features/newUserRegistration.feature]: may be use `data table` instead :+1:

Comment (susmitathapa) [reply Features5/tests/features/newUserRegistration.feature]: -> Those are for different scenarios, do we make data table for different scenarios too?

File: Features5/tests/features/newUserRegistration.feature [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+   
+   Scenario: New user registration with valid inputs
+       When user enters following valid credentials:
+         |name    |email                |password   |confirmPassword  |
+         |Susmita |susmita412@gmail.com |zxcvbnm    |zxcvbnm          |
+       And clicks "Register" button
+       Then user should see the homepage
+
+    Scenario Outline: When new user registers with invalid inputs
+       When register with foloowing invalid inputs and clicks "Register" button
```
Comment (SagarGi) [inline Features5/tests/features/newUserRegistration.feature]:
```suggestion
       When the user registers with the following invalid inputs
```
we can add button click to our code and no need to specify on steps

**Review by SagarGi — APPROVED** [review]

**Pull request merged**